app/views/shared/components/_tags_list.html.slim

```diff
@@ -0,0 +1,6 @@
+p
+  span.glyphicon.glyphicon-tag.glyphicon--right
+  - tags.each_with_index do |tag, i|
+    = link_to tag.name, root_path
```

Comment:
Oh, I thought we would have separate pages for tags, which is why I left it like this, but linking it to the search is fine as well.